### PR TITLE
Get latest extension state during refresh

### PR DESCRIPTION
**Pull request by ravipal**
vs code docker extension calls refresh on the Azure Registry node which needs to reflect the latest extension state.

### CODE REVIEW

File: ui/src/treeDataProvider/AzureAccountTreeItemBase.ts [hunk 1]
```diff
@@ -59,6 +61,8 @@ export abstract class AzureAccountTreeItemBase extends AzExtParentTreeItem imple
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean, context: types.IActionContext): Promise<AzExtTreeItem[]> {
+        // Refresh the AzureAccount, to handle the Azure account extension installed/uninstalled between refreshes
+        this._azureAccountTask = this.loadAzureAccount(this._testAccount);
```
Comment (EricJizbaMSFT) [inline ui/src/treeDataProvider/AzureAccountTreeItemBase.ts]: I believe this means `loadAzureAccount` will subscribe to some events multiple times. Can you double check and make sure we only subscribe to those events once

File: ui/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureextensionui",
     "author": "Microsoft Corporation",
-    "version": "0.29.0",
+    "version": "0.30.0",
```
Comment (EricJizbaMSFT) [inline ui/package.json]: tl;dr please update to 0.29.1 instead. Also update the version in package-lock.json

More explanation:
Normally, you're [supposed to bump](https://semver.org/) the major version for breaking changes. Since we're not ready to commit to "1.0.0" yet, we shift the meaning of each position down. Thus we bump the minor version for breaking changes and the patch version for anything else. Since you did not introduce any breaking changes, you don't need to bump the minor version.

File: ui/src/treeDataProvider/AzureAccountTreeItemBase.ts (hunk 1 above)
Comment (ravipal) [reply ui/src/treeDataProvider/AzureAccountTreeItemBase.ts]: Now I call the loadAzureAccount only when the old account is null.

File: ui/package.json (hunk 2 above)
Comment (ravipal) [reply ui/package.json]: Thanks! Good to know.
